vault/logical_system.go

```diff
@@ -1498,6 +1501,23 @@ func (b *SystemBackend) handleMount(ctx context.Context, req *logical.Request, d
 		config.DelegatedAuthAccessors = apiConfig.DelegatedAuthAccessors
 	}
 
+	if apiConfig.IdentityTokenKey != "" {
+		config.IdentityTokenKey = apiConfig.IdentityTokenKey
```

Comment:
nit: should we only set the Identity Token Key to the config after we can validate that it exists below?

Reply:
Can switch the ordering to check first 👍

Reply:
Changed ordering!